doc/source/operations/cloudkitty.rst

```diff
@@ -0,0 +1,125 @@
+==========
+CloudKitty
+==========
+
+Configuring in kayobe-config
+============================
+
+By default, CloudKitty uses Gnocci and Ceilometer as the collector and fetcher
+backends. Unless the system has a specific reason not to, we recommend instead
+using Prometheus as the backend for both. The following instructions explain
+how to do this
+
+Enable CloudKitty and disable InfluxDB, as we are using OpenSearch as the
+storage backend. Set the following in ``kolla.yml``:
+
+.. code-block:: console
+
+  kolla_enable_cloudkitty: true
+  # Explicitly disable influxdb as we are using OpenSearch as the CloudKitty backend
+  kolla_enable_influxdb: false
+
+Use Prometheus as the collector and fetcher backends, and Elasticsearch as the
+storage backend. Note that this is patched in our fork of CloudKitty to also
```

Comment:
Don't we exclusively use `OpenSearch`, as well as we aren't able to set `cloudkitty_storage_backend: opensearch` as the backend, as it won’t work? This is to say that this sentence suggests that "opensearch" can be set for `cloudkitty_storage_backend:` when, in fact, nothing needs changing if you want to use `opensearch`.

Maybe something like: In our fork of CloudKitty `cloudkitty_storage_backend: elasticsearch` is patched to correspond to both ElasticSearch as well as OpenSearch.

Reply:
Discussed this in more detail with Max in person, will update this section to make things clearer.